playbook/app/pb_kits/playbook/pb_filter/Filter/FilterDouble.jsx

```diff
@@ -18,16 +18,22 @@ export type FilterDoubleProps = {
   sortValue?: SortValue,
 } & FilterBackgroundProps
 
-const FilterDouble = ({ onSortChange, sortOptions, sortValue, filters, results, children, ...bgProps }: FilterDoubleProps) => (
-  <FilterBackground {...bgProps}>
+const FilterDouble = ({ onSortChange, sortOptions, sortValue, filters, results, children, dark, ...bgProps }: FilterDoubleProps) => (
+  <FilterBackground
+      dark={dark}
+      {...bgProps}
+  >
     <Flex
         orientation="row"
         vertical="center"
     >
-      <FiltersPopover>
+      <FiltersPopover dark={dark}>
         {children}
       </FiltersPopover>
-      <CurrentFilters filters={filters} />
+      <CurrentFilters
+          dark={dark}
+          filters={filters}
+      />
     </Flex>
     <SectionSeparator />
```

Comment:
@adub65 Think you'll need to add another `dark={dark}` here on the SectionSeparator.